app/utils.py

```diff
@@ -127,12 +127,36 @@ def email_address_is_nhs(email_address):
     ))
 
 
-def update_dct_to_str(update_dct):
-    str = "\n"
+def update_dct_to_str(update_dct, lang):
+    if lang not in ['EN', 'FR']:
+        raise NotImplementedError
+
+    values = {
+        'EN': {
```

Comment:
Have a look in rest.py - there's a set_permissions() function there that also sends a change alert when a users' service permissions change. There may be some extra values needed for that.

Reply:
Here is the current email sent when changing permissions. I guess it's not ideal but I don't see how we could improve it easily. The 🇫🇷   part is correct from a copywriting perspective

![Screen Shot 2020-09-17 at 11 39 47](https://user-images.githubusercontent.com/295709/93494032-91268080-f8da-11ea-9fa3-25a794dc9b85.png)